guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: guide
+title: Transaction privacy
+description: An overview of how to help users maintain their financial privacy while using Bitcoin.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/privacy/privacy-preview.jpg
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/privacy/header.jpg"
+    retina = "/assets/images/guide/payments/privacy/header@2x.jpg"
+    mobile = "/assets/images/guide/payments/privacy/privacy-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/privacy/privacy-header-mobile@2x.jpg"
+    alt-text = "Illustration of an eye, with a cross ontop of an ink blot background"
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+
+# Transaction privacy
+
+<!--
+This page should inform about what information is made public when sending or receiving, how the application can help minimize unnecessary privacy leaks, basic application functionality to help, and when we can, more advanced options.
+- Single-use addresses (avoiding address reuse)
+- Coin selection / labelling
+- Coin join / Pay join
+-->
+
+It’s a common misconception that Bitcoin is anonymous. Instead, it is referred to as pseudonymous which means each address you generate is like a new identity. Unless you reveal that you are the owner whether directly or indirectly you are able to remain anonymous.
+
+When mined, transactions, their signatures, and addresses are public on the ledger, forever. There is no personally identifiable information attached to transactions on the blockchain though. The transparent property of the blockchain allows anyone to easily verify the amount of Bitcoin in existence but also makes looking up any address or transaction trivial. This can be demonstrated going back to the first block on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b), a transaction mined by Satoshi Nakamoto.
+
+> Each Bitcoin transaction contains at least one input and at least one output. This means that once a single address is known, there is a trail to follow the Bitcoin.
+> 
+> <cite>As documented by <a href="https://docs.wasabiwallet.io/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-is-bitcoin-bad-in-terms-of-privacy">Wasabi Wallet</a></cite>
+
+The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know that they mined the first block, which contained just one transaction, the block reward sent to their address. This illustrates the permanence of the association to an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. Once an address owner's identity is made public it becomes a difficult task to break the connection.
```

Comment:
```suggestion
The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know that they mined the first block, which contained just one transaction, with the block reward getting sent to their address. This illustrates the permanence of the association to an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. Once an address owner's identity is made public it becomes a difficult task to break the connection.
```

Reply:
Good grammar suggestions, commit.